node/sc/bridge_manager.go

```diff
@@ -313,7 +313,7 @@ func (bi *BridgeInfo) handleRequestValueTransferEvent(ev *RequestValueTransferEv
 		uri, err = erc721.TokenURI(nil, ev.ValueOrTokenId)
 		if err != nil {
 			if err == vm.ErrExecutionReverted {
-				logger.Warn("The ERC721 token does not support URI", "erc721", ev.TokenAddress.String(), "onParent", bi.onChildChain, "tokenId", ev.ValueOrTokenId.String())
+				logger.Debug("Unable to get an ERC721 URI", "erc721", ev.TokenAddress.String(), "onParent", bi.onChildChain, "tokenId", ev.ValueOrTokenId.String())
```

Comment:
How about `Trace()`?

Reply:
Good. I will modify this to `trace` if bridge contract is not mint/burn mode. :)